sdk/lib/rtl/debug.c

```diff
@@ -125,6 +125,7 @@ vDbgPrintExWithPrefixInternal(IN PCCH Prefix,
         ExceptionRecord.ExceptionRecord = NULL;
         ExceptionRecord.NumberParameters = 2;
         ExceptionRecord.ExceptionFlags = 0;
+        /* FIXME: '+ 1' looks wrong when Length == sizeof(Buffer) */
```

Comment:
+ 1 is correct here. The buffer should always be NULL terminated and Length should be less than sizeof(Buffer). There is a bug in the overflow code above though, which does not NULL terminate the buffer, but instead only writes a '\n'.
Please remove this comment, I wil address the issue in a seprarate PR.